pkg/consensus/consensus.go

```diff
@@ -495,3 +496,30 @@ func convertKeys(validators []crypto.PublicKey) (pubs []*keys.PublicKey) {
 
 	return
 }
+
+func newBlockFromContext(ctx *dbft.Context) block.Block {
+	block := new(neoBlock)
+	if ctx.TransactionHashes == nil {
+		return nil
+	}
+
+	block.SetTimestamp(ctx.Timestamp)
+	block.SetIndex(ctx.BlockIndex)
+	block.SetNextConsensus(ctx.NextConsensus)
+	block.SetPrevHash(ctx.PrevHash)
+	block.SetVersion(ctx.Version)
+	block.SetConsensusData(ctx.Nonce)
```

Comment:
You don't need any of these setters. They can actually be dropped now. `neoBlock` is completely yours here.

Reply:
Changed this to direct assignment. Do I have to completely remove these setters from both `neo-go` and `dbft`?

Reply:
I think so, new interface eliminates the need for them.